templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -68,7 +74,15 @@ Parameters:
   KeyPairName:
     Description: Public/private key pairs allow you to securely connect to your instance
```

Comment:
Can we change this to:
> Public/private EC2 Key Pairs to allow you to securely access the Bastion host

Reply:
I'd be inclined to leave the wording on this one, its for the `quickstart-vpc-for-atlassian-services.yaml ` so QS users wont see it, and furthermore the key is potentially used by upstream templates for other purposes than bastions

Reply:
I don't have a strong opinion on wording, both seem fine to me I just wanted to note:

I don't think other templates are using `KeyPairName` for other purposes. And if they do, we would need to revisit our decision to make them optional. The application nodes were created with this KeyPairName so the Bastion node was able to jump on the with the same ssh key. The VPC template was using the KeyPairName for NAT instances but they are deprecated as the NAT gateways are now supported everywhere:

```
KeyPairName:
    Description: Deprecated. NAT gateways are now supported in all regions.
```

Reply:
It was the VPC template that I was referring to when I was of the impression that `KeyPairName` was used for other purposes. I wasnt sure whether to trust the `deprecation` flags in that template hence my comment. This is good to know @nanux and that`KeyPairName` is specifically for bastions.